docs/src/understand/thin-edge-json.md

```diff
@@ -1,52 +1,107 @@
+subscribing to various MQTT topics.
+Any data can be forwarded to the connected cloud-provider as well, by publishing the data to some standard topics.
+
+### Sending Telemetry Data to thin-edge.io
+
+All topics with the prefix `tedge/` are reserved by `thin-edge.io` for this purpose.
+To send measurements to `thin-edge.io`, the measurements represented in Thin Edge JSON format can be published
+to the `tedge/measurements` topic.
+Other processes running on the thin-edge device can subscribe to this topic to process these measurements.
+
+If the messages published to this `tedge/measurements` topic is not a well-formed Thin Edge JSON,
+then that message won’t be processed by `thin-edge.io`, not even partially,
+and an appropriate error message on why the validation failed will be published to a dedicated `tedge/errors` topic.
+The messages published to this topic will be highly verbose error messages and can be used for any debugging during development.
+You should not rely on the structure of these error messages to automate any actions as they are purely textual data
+and bound to change from time-to-time.
+
+Here is the complete list of topics reserved by `thin-edge.io` for its internal working:
+
+| Topic                                                | Description                                                            |
+|------------------------------------------------------|------------------------------------------------------------------------|
+| `tedge/`                                             | Reserved root topic of `thin-edge.io`                                  |
+| `tedge/measurements`                                 | Measurements related to the main device                                |
+| `tedge/measurements/${child-id}`                     | Measurements related to the child device named `${child-id}`           |
+| `tedge/events/${event-type}`                         | Events related to the main device                                      |
+| `tedge/events/${event-type/${child-id}`              | Events related to the child device named `${child-id}`                 |
+| `tedge/alarms/${severity}/${alarm-type}`             | Alarms related to the main device                                      |
+| `tedge/alarms/${severity}/${alarm-type}/${child-id}` | Alarms related to the child device named `${child-id}`                 |
+| `tedge/errors`                                       | Error messages emitted by `thin-edge.io` while processing measurements |
+
+### Sending Telemetry Data to the cloud
+
+The `thin-edge.io` framework allows users forward telemetry data generated and published to one of the
+`tedge/#` MQTT topics from the thin-edge device to any IoT cloud provider that it is connected to,
+with the help of a *mapper* component designed for that cloud.
+The responsibility of a mapper is to subscribe to the `tedge/#` topic to receive all incoming measurements, events and alarms
```

Comment:
```suggestion
The responsibility of a mapper is to subscribe to the `tedge/#` topic to receive all incoming data
```
Just to be broader to cover operations data as well

Reply:
Updated